packages/components/addon/components/hds/stepper/indicator/task.js

```diff
@@ -35,6 +34,29 @@ export default class HdsStepperIndicatorTaskIndexComponent extends Component {
     return this.args.isInteractive || false;
   }
 
+  /**
+   * @param iconName
+   * @type {string}
+   */
+
+  get iconName() {
+    let status = this.args.status;
```

Comment:
[issue] Done in this way you are missing the default value (`DEFAULT_STATUS`) because you're reading the argument. If you instead you use `this.status` is already taken in account for you by the component class itself (see code example below).